packages/eslint-plugin/src/configs/react.js

```diff
@@ -58,5 +58,11 @@ module.exports = {
         'react/jsx-no-bind': 'off',
       },
     },
+    {
```

Comment:
we cannot do this as anyone consuming this outside our repo would get Exceptions.

IN order to use workspace lint rules right now we need to follow `__internal` pattern

see `packages/eslint-plugin/src/internal.js`

```js
const __internal = {
  /**
   * `@nx/eslint-plugin` is necessary in order to register custom lint rules that live within tools/eslint-rules
   */
  plugins: shouldRegister ? ['@nx'] : [],
// extend this object with your rule overrides
  overrides: {
    react: shouldRegister
      ? {
          files: ['**/src/**/*.{ts,tsx}'],
          rules: {
            '@nx/workspace-consistent-callback-type': 'error',
          },
        }
      : null,
  },
};
```

then we can use it here like :

```
{
      files: ['**/*.cy.{ts,tsx,js}', 'isConformant.{ts,tsx,js}'],
      rules: {
        'import/no-extraneous-dependencies': 'off',
        'react/jsx-no-bind': 'off',
      },
    },
 // 1. our internal overrides
    __internal.overrides.react,
  // 2. lets get rid of `null` for public api consumers
  ].filter(Boolean),
```